lib/puppet/parser/functions/consul_sorted_json.rb

```diff
@@ -6,11 +6,13 @@ class << self
 
     def sorted_generate(obj)
       case obj
-        when Fixnum, Float, TrueClass, FalseClass, NilClass
-          return obj.to_json
+        when NilClass
+          return "null"
```

Comment:
We just need to ensure that it is returning actual null, not the string "null" as those are in fact 2 different things in the JSON spec.